commands/guild/welcome.js

```diff
@@ -0,0 +1,39 @@
+const { Command } = require('discord.js-commando');
+const db = require('quick.db');
+
+module.exports = class WecomeMessageCommand extends Command {
+  constructor(client) {
+    super(client, {
+      name: 'welcome-message',
+      memberName: 'welcome-message',
+      aliases: ['welcomemessage', 'welcome'],
+      group: 'guild',
+      guildOnly: true,
+      clientPermissions: ['ADMINISTRATOR'],
+      description: 'Allows you to toggle the welcome message for new members that join the server.',
+      args: [
+        {
+          key: 'choice',
+          prompt:
+            'Do you want me to welcome new members? Type Yes or No',
+          type: 'string',
+          oneOf: ['yes', 'no', 'enable', 'disable']
+        }
+      ]
+    });
+  }
+
+  run(message, { choice }) {
+    if (choice.toLowerCase() == 'enable') var choice = 'yes';
```

Comment:
you can just change the value stored in 'choice' instead of creating a new variable using the 'var' keyword